app/scripts/views/mixins/signup-mixin.js

```diff
@@ -31,6 +31,8 @@ define(function (require, exports, module) {
      */
     signUp: function (account, password) {
       var self = this;
+      self.logEvent('flow.signup.submit');
```

Comment:
> How will users that sign-in from the /signup page affect this stat?

Yep, just to confirm @vladikoff's answer, that is by design as per the feature doc.